services/port_manager/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.6.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.futurewei.alcor</groupId>
+    <artifactId>AlcorPortManager</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>AlcorPortManager</name>
+    <description>Alcor Port Manager Module</description>
+
+    <properties>
+        <java.version>1.8</java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.futurewei.alcor.common</groupId>
+            <artifactId>AlcorCommonLib</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.futurewei.alcor.web</groupId>
+            <artifactId>AlcorWeb</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
Let us turn on port manger UTs.

Reply:
The UT of port manager need to interact with many microservices. If it is opened, the test will fail. Do you have any suggestions?

Reply:
@chenpiaoping SubnetManager UTs has similar issues previously and used Mockito to solve this problem. Check this out:

https://github.com/futurewei-cloud/alcor/blob/master/services/subnet_manager/src/test/java/com/futurewei/alcor/subnet/SubnetControllerTests.java

Reply:
@chenpiaoping I will complete reviewing the rest of PR by tomorrow. Meanwhile please use the time to turn on UTs and address existing comments.

Reply:
Okay, thanks xieus.